dashboard/pages/primary_screening/callbacks.py

```diff
@@ -273,8 +276,28 @@ def on_summary_entry(current_stage: int, stored_uuid: str, file_storage: FileSto
 
     echo_bmg_combined = echo_bmg_combined.reset_index().to_dict("records")
```

Comment:
As of current this line crashes the server when run twice (on changed echo_df - e.g. when user goes back from `report` to `summary` or does not reupload the echo files)